README.md

```diff
@@ -65,7 +65,7 @@ There are some guidelines which will make applying PRs easier for us:
  Optional supplemental description.
 ```
 + Make sure you have added the necessary tests (JUnit/IT) for your changes.
-+ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken.
++ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you may need to run `mvn install` beforehand).
```

Comment:
Review hint: This was not introduced by @rmannibucau’s change, mrm+invoker was already broken before.